data-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/data/common/TestSerDesConfig.java

```diff
@@ -45,7 +45,7 @@ private TestSerDesConfig() {
      *
      * @return an appropriate key deserialiser for the topic's message content
      */
-    public static Deserializer<String> keyDeserializer() {
+    public static Deserializer<String> keyDeserialiser() {
```

Comment:
Can this be changed to `successKeyDeserialiser`?
That way it conforms to how the error deserialiser is named

Reply:
Please check this as it does not appear to have been addressed and on line 57

Reply:
Ok, done.